### DISCUSSION TITLE
opensnitch dependencies...

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1230] [Q&A] [answered]
Hi,

I've read 
- [Rpi4 4gb - Tutorial to install : Daemon, Gui - Armh version](https://github.com/gustavo-iniguez-goya/opensnitch/issues/42)
- [Wiki: Dependencies-and-how-it-works](https://github.com/evilsocket/opensnitch/wiki/Dependencies-and-how-it-works)

So (for system with nftables ) the dependencies are :
- nftables (obvisouly)
- libnetfilter-queue1
- /proc directory tree, or eBPF

Am I correct?

Thanks.

**Comment by BobSquarePants** [comment, marked as answer]
If done some digging and
```bash
$dpkg-deb -f /path/to/opensnitch_1.6.6-1_armhf.deb Depends
libnetfilter-queue1, libc6, libnfnetlink0
```

**Reply by gustavo-iniguez-goya** [reply]
yeah, basically those libs and procfs mounted for the daemon.

**Reply by BobSquarePants** [reply]
Thank you @gustavo-iniguez-goya , out of topic, when a discussion has been concluded, do you prefer that OP[^1] ( close, let open , don't matter ) the discussions ?

Thanks.


[^1]: original poster
